### PR TITLE
add modules

### PR DESCRIPTION
**Pull request by SSEKPIUS**
In this project, I will restructure my Awesome books app code. The goal is to make it more organized by using modules.

### CODE REVIEW

File: modules/dom_methods.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line import/no-mutable-exports
+let books = [];
+
```
Comment (Wes-Isaac) [inline modules/dom_methods.js]: - [ ] Great job, please don't disable linters, it better to solve the issues that come up rather than disabling them 👍.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @SSEKPIUS, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter error 👍.
- Professional README 💯.
- Use of ES6 features 🥇.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: modules/dom_methods.js (hunk 1 above)
Comment (SSEKPIUS) [reply modules/dom_methods.js]: @Wes-Isaac , I had trouble overcoming that

**Review by gemmen29 — APPROVED** [review]
Hi @SSEKPIUS, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**